aws/endpoints/defaults.go

```diff
@@ -24300,6 +24309,15 @@ var awsusgovPartition = partition{
 						Region: "us-gov-west-1",
 					},
 				},
+				endpointKey{
+					Region:  "us-gov-west-1",
+					Variant: fipsVariant,
+				}: endpoint{
+					Hostname: "ec2.us-gov-east-1.amazonaws.com",
```

Comment:
should be "ec2.us-gov-west-1.amazonaws.com" ?

Reply:
Fixed.